lib/huginn_agent.rb

```diff
@@ -1,17 +1,45 @@
 require 'huginn_agent/version'
 
 class HuginnAgent
```

Comment:
Not sure if it is a good idea to use the same namespace as the Huginn rails application. Depending on the load order we would either monkey path the rails application or the other way around.